chat-plugins/jeopardy.js

```diff
@@ -1,772 +1,746 @@
+			if (reply) {
+				this.errorReply(reply);
+			} else {
+				this.sendReply("Questions have been imported.");
+			}
+		},
+
+		dailydouble: 'dd',
+		dd: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
+			let params = target.split(",");
+			if (params.length !== 2) return this.errorReply("You must specify the category number and question number");
+			let categoryNumber = parseInt(params[0]);
+			if (!categoryNumber || categoryNumber < 1 || categoryNumber > room.game.categoryCount) return this.errorReply(`The category must be a number between 1 and ${room.game.categoryCount}.`);
+			let questionNumber = parseInt(params[0]);
+			if (!questionNumber || questionNumber < 1 || questionNumber > room.game.questionCount) return this.errorReply(`The question must be a number between 1 and ${room.game.questionCount}.`);
+			let reply = room.game.setDailyDouble(categoryNumber - 1, questionNumber - 1);
+			if (reply) {
+				this.errorReply(reply);
+			} else {
+				this.sendReply("Daily double has been added.");
+			}
+		},
+
+		dailydoublehelp: ["/jeopardy dailydouble [category number], [question number] - Set a question to be a daily double."],
+		view: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
+			let params = target.split(",");
+			if (params.length !== 2) return this.errorReply("You must specify the category number and question number");
+			let categoryNumber = parseInt(params[0]);
+			if (!categoryNumber || categoryNumber < 1 || categoryNumber > room.game.categoryCount) return this.errorReply(`The category must be a number between 1 and ${room.game.categoryCount}.`);
+			let questionNumber = parseInt(params[1]);
+			if (!questionNumber || questionNumber < 1 || questionNumber > room.game.questionCount) return this.errorReply(`The question must be a number between 1 and ${room.game.questionCount}.`);
+			this.sendReplyBox(room.game.getQuestion(categoryNumber - 1, questionNumber - 1));
+		},
+
+		addplayer: 'adduser',
+		adduser: function (target, room, user) {
+			if (!room.game || room.game.gameid !== 'jeopardy') return this.errorReply("There is no game of Jeopardy going on in this room.");
+			if (user.userid !== room.game.host.userid) return this.errorReply("This command can only be used by the host.");
+			let targetUser = Users.get(target);
+			if (!targetUser) return this.errorReply("User '" + target + "' not found.");
+			if (room.game.addPlayer(targetUser)) {
```

Comment:
I don't think hosts should be able to make themselves players of their own games